cli/src/error.rs

```diff
@@ -55,7 +55,7 @@ impl std::fmt::Display for CliError {
             CliError::User(ref s) => write!(f, "Error: {}", s),
             CliError::Io(ref err) => write!(f, "IoError: {}", err),
             CliError::Signing(ref msg) => write!(f, "SigningError: {}", msg),
-            CliError::Hyper(ref err) => write!(f, "HyperError: {}", err),
+            CliError::Request(ref err) => write!(f, "ReqwestError: {}", err),
```

Comment:
This should be Request instead of Reqwest

Reply:
Fixed, thanks